nym-vpnd/src/service/error.rs

```diff
@@ -0,0 +1,123 @@
+use std::path::PathBuf;
+
+use nym_vpn_lib::credential_storage::error::StorageError;
+use nym_vpn_lib::credentials::CredentialError;
+use nym_vpn_lib::id::NymIdError;
+use time::OffsetDateTime;
+use tracing::error;
+
+#[derive(Clone, Debug, thiserror::Error)]
+pub enum ImportCredentialError {
+    #[error("vpn is connected")]
+    VpnRunning,
+
+    #[error("credential already imported")]
+    CredentialAlreadyImported,
+
+    #[error("generic error: {0}")]
+    Generic(String),
```

Comment:
that's a bit obscure error enum, but I guess we gotta do what we gotta do.
I just have a personal bias against error enum variants that just wrap around a string, but I recognise sometimes we have no better options and I think this might be one of those instances